sagemaker-jumpstart/object_detection/JumpStart_Object_Detection.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.        region=None,
</pre><p>Do you want to pass in the current region rather than specifying None?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>This is also handled at the backend.</p>